docs/guide/cli/index.md

```diff
@@ -0,0 +1,253 @@
+# CLI
+
+The e18e CLI is a powerful tool for analyzing and optimizing your JavaScript/TypeScript projects. It helps you identify performance issues, find optimization opportunities, and migrate to better alternatives.
```

Comment:
```suggestion
The e18e CLI is a powerful tool for analyzing and optimizing your JavaScript/TypeScript projects. It helps you identify performance issues, find optimization opportunities, and automatically migrate flagged code
```

maybe this will be better since it may not forever just be migration to alternative packages